packages/cli/src/commands/logout.ts

```diff
@@ -76,25 +89,10 @@ export default async function main(client: Client): Promise<number> {
     output.debug('Configuration has been deleted');
   } catch (err) {
     output.error(`Couldn't remove config while logging out`);
-    return 1;
-  }
-
-  const res = await fetch(`${apiUrl}/v3/user/tokens/current`, {
-    method: 'DELETE',
-    headers: {
-      Authorization: `Bearer ${token}`,
-    },
-  });
-
-  if (res.status === 403) {
-    output.debug('Token is invalid so it cannot be revoked');
-  } else if (res.status !== 200) {
-    const err = await res.json();
-    output.error('Failed to revoke token');
-    output.debug(err ? err.message : '');
-    return 1;
+    output.debug(err?.message ?? '');
+    exitCode = 1;
```

Comment:
I think we want to return early still here, otherwise "Logged out!" success message is gonna be printed.

Reply:
@TooTallNate We can't return early because we need to make sure that the file IO happens even if the API is down so that we don't even prevent users from logging out

Reply:
@TooTallNate I see what you mean now. I updated the "Logged out" message so it will not print unless the exit code is `0`